src/main/java/com/nkh/ECommerceShop/controller/AuthController.java

```diff
@@ -53,10 +60,26 @@ public ResponseEntity<?> authenticateUser(@RequestBody @Valid UserCredentialsDTO
         ResponseCookie jwtCookie = jwtUtils.generateJwtCookie(userDetails);
 
         List<String> roles = userDetails.getAuthorities().stream()
-                .map(item -> item.getAuthority())
-                .collect(Collectors.toList());
+                .map(GrantedAuthority::getAuthority)
+                .toList();
+        RefreshToken refreshToken = refreshTokenService.createRefreshToken(userDetails.getId());
```

Comment:
this is not used?

Reply:
it was suggestion from IDE to use just toList() instead of collect(Collectors.toList())